services/generator/service.go

```diff
@@ -464,6 +465,10 @@ func (s *service) GenerateTemplateData(params api.Params, currentReplicas int, g
 	if data.UseGCEIngress && !strings.HasSuffix(data.IngressPath, "*") {
 		data.IngressPath += "*"
 	}
+	data.PathType = "Prefix"
```

Comment:
Do this on line 57 when instantiating the `data` object.

Reply:
Done